jerry-core/ecma/builtin-objects/ecma-builtin-object.c

```diff
@@ -135,6 +135,127 @@ ecma_builtin_object_object_get_prototype_of (ecma_value_t this_arg, /**< 'this'
   return ret_value;
 } /* ecma_builtin_object_object_get_prototype_of */
 
+#ifndef CONFIG_DISABLE_ES2015_BUILTIN
```

Comment:
IMHO, this is too general name. It would be better to use something like `CONFIG_DISABLE_ES2015_BUILTIN_COMMON` or `CONFIG_DISABLE_ES2015_BUILTIN_OBJECT`